linuxdeploy-plugin-gtk.sh

```diff
@@ -124,12 +129,18 @@ echo "Copying more libraries"
 gobject_libdir="$("$PKG_CONFIG" --variable=libdir gobject-2.0)"
 gio_libdir="$("$PKG_CONFIG" --variable=libdir gio-2.0)"
 librsvg_libdir="$("$PKG_CONFIG" --variable=libdir librsvg-2.0)"
-cp $verbose \
-    "$gdk_libdir/"libgdk_pixbuf*.so* \
-    "$gobject_libdir/"libgobject*.so* \
-    "$gio_libdir/"libgio*.so* \
-    "$librsvg_libdir/"librsvg*.so* \
-    "$APPDIR/usr/lib/"
-cat >> "$HOOKFILE" <<EOF
-export LD_LIBRARY_PATH="\$APPDIR/usr/lib:\$LD_LIBRARY_PATH"
-EOF
+FIND_ARRAY=(
+    "$gdk_libdir"     "libgdk_pixbuf-*.so*"
+    "$gobject_libdir" "libgobject-*.so*"
+    "$gio_libdir"     "libgio-*.so*"
+    "$librsvg_libdir" "librsvg-*.so*"
+)
+LIBRARIES=()
+for (( i=0; i<${#FIND_ARRAY[@]}; i+=2 )); do
+    directory=${FIND_ARRAY[i]}
+    library=${FIND_ARRAY[i+1]}
+    while IFS= read -r -d '' file; do
+        LIBRARIES+=(--library="$file")
+    done < <(find "$directory" -type f -name "$library" -print0)
```

Comment:
This form is recommended by ShellCheck (see [SC2044](https://github.com/koalaman/shellcheck/wiki/SC2044)).
A loop over `find` is better than a `| head -n1` in my point of view.

Reply:
Retrospectively, it might have been easier to just `cp` the files in and then have linuxdeploy run over the AppDir. linuxdeploy will anyway redeploy all binaries in there. `-l` wouldn't strictly be necessary.

Nevertheless, this looks pretty solid, so let's go for this approach.